test/data/TestClient/lib/testClient.ts

```diff
@@ -12,10 +12,10 @@
 /* jshint forin:false */
 /* jshint noempty:false */
 
-'use strict';
+"use strict";
```

Comment:
`--alwaysStrict` which is implied by `--strict` will always put "use strict" in the output .js file. No need to go on a hunt to remove them all though.